backend/dataInterface/mongoDB.js

```diff
@@ -79,8 +100,181 @@ module.exports.getFireCommentsByFire = async (id) => {
   } finally {
     await client.close();
   }
+};
+
+//create a comment by fireid, userid
+module.exports.createComment = async (paramObj, newObj) => {
+  try {
+    await client.connect();
+    const db = client.db(databaseName);
+    const collection = db.collection(comments_Collection);
+    const commentObj = {
+      ...newObj,
+      fire_id: ObjectId(paramObj.id),
+      username: paramObj.userName,
+      text: newObj.text,
```

Comment:
Would this field already be created by the `...newObj` on line 112?

Reply:
Leave as is for now.  This is to attach newly created comment to an array of existing comments created for a fireid